static/css/zamboni/zamboni.css

```diff
@@ -2020,6 +2020,7 @@ form .error .note.error {
     -moz-border-radius: 5px;
     -webkit-border-radius: 5px;
     border-radius: 5px;
+    width: 95%;
```

Comment:
You should be able to set the width to `100%` and use [`word-wrap: break-word;`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) so the `<blockquote>` doesn't have less width than intended. Let me know if that works; I think it would be a bit nicer.